app/src/main/res/layout/app_version_activity.xml

```diff
@@ -29,13 +29,22 @@
       app:titleTextColor="@color/white" />
   </com.google.android.material.appbar.AppBarLayout>
 
+  <View
+    android:id="@+id/app_version_shadow_view"
+    android:layout_width="match_parent"
```

Comment:
Make this `0dp` and add
`start-to-start-of = parent`
`end-to-end-of = parent`

constraints too.

Reply:
Done